src/main/java/uk/gov/hmcts/cmc/claimstore/events/DocumentGeneratedEvent.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.claimstore.events;
+
+import uk.gov.hmcts.cmc.claimstore.documents.output.PDF;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.List;
+
+import static com.google.common.collect.Lists.newArrayList;
+
+public class DocumentGeneratedEvent {
+    private final Claim claim;
+    private final String authorization;
+    private final List<PDF> documents;
+
+    public DocumentGeneratedEvent(final Claim claim, String authorization, PDF... documents) {
+        this(claim, authorization, newArrayList(documents));
+    }
+
+    public DocumentGeneratedEvent(final Claim claim, String authorization, final List<PDF> documents) {
+        this.claim = claim;
+        this.documents = documents;
```

Comment:
can you sync the order of parameters to fields you are assigning please?

Reply:
Well spotted. Done.